source/_integrations/neato.markdown

```diff
@@ -24,40 +25,42 @@ There is support for the following platform types within Home Assistant:
 - **Switch** - allows you to enable or disable the schedule.
 - [**Vacuum**](#vacuum)
 
-To activate `neato` in your installation, you can set it up from the integration screen or add it to your `configuration.yaml` file.
+## Configuration
 
-## Setup the integration via the integrations screen
+1. Visit [the Neato Developer Network](https://developers.neatorobotics.com/applications) and create a new app.
```

Comment:
Lets make sure to mention the user needs to check all 3 scopes for the items below:

![image](https://user-images.githubusercontent.com/1634145/101953236-e0651000-3bae-11eb-88b9-965a1fde053f.png)

Reply:
Done. Thanks :)